tools/busywork/tcl/fabric.tcl

```diff
@@ -357,7 +357,7 @@ proc ::fabric::caLogin {i_peer i_user i_secret} {
     http::cleanup $token
 }
 
-    
```

Comment:
@bcbrock I apologize for all the whitespace churn here, I didn't notice it happened in my preliminary reviews.  I think emacs got a little overzealous on me.  If this was not the right thing to do, I can revert judiciously apply the meaningful hunks.